package.json

```diff
@@ -10,6 +10,7 @@
     "concurrently": "^6.2.0",
     "eslint": "^7.30.0",
     "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-jsdoc": "^35.5.0",
```

Comment:
Can we use tsdoc instead of jsdoc? Not super familiar with the tradeoffs, but if tsdoc is more Typescript-compatible it seems better? https://tsdoc.org/pages/packages/eslint-plugin-tsdoc/

Reply:
Yes, we should, as I believe the interpreter will behave more consistently and take static typing into account

Reply:
I originally did, but there's no way to enforce that things are actually documented with TSDoc which is *super* annoying